scalafix-sbt/src/sbt-test/sbt-scalafix/cross-build/javaProject/src/main/java/example/Example.java

```diff
@@ -0,0 +1,7 @@
+package example;
+
+class Foo  {
```

Comment:
can you add a test like: https://github.com/scalacenter/scalafix/issues/331#issuecomment-327733840

> This may be related to the fact that in my project the integration test .scala files are in a sub-folder of the test/ source directory, e.g.:

Reply:
Given the implementation uses directly `unmanagedSources` I think it would be an sbt bug if that failed now.